python/helper-image/launcher/cmd_test.go

```diff
@@ -0,0 +1,155 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"os/exec"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+)
+
+func TestCmd(t *testing.T) {
+	RunCmdOut([]string{"hello"}, "abc").
+		AndRunCmdFail([]string{"ls"}, 1).
+		Setup(t)
+
+	if out, err := newCommand(nil, []string{"hello"}, nil).Output(); err != nil {
+		t.Error("command should not have failed")
+	} else if string(out) != "abc" {
+		t.Error("output should have been abc")
+	}
+	if newCommand(nil, []string{"ls"}, nil).Run() == nil {
+		t.Error("command should have failed")
+	}
+}
+
+type fakeCmd struct {
+	mode     string
+	cmdline  []string
+	exitCode int
+	output   string
+}
+
+var _ commander = (*fakeCmd)(nil)
```

Comment:
nit: might be helpful to add a comment that this is check verifying that fakeCmd satisfies the `commander` interface